backend/app/Http/Controllers/TransactionController.php

```diff
@@ -11,6 +11,38 @@
 
 class TransactionController extends Controller
 {
+    public static function creditTransaction(Account $account, $payload)
```

Comment:
```suggestion
    public static? function creditTransaction(Account $account, $payload)
```
is there a need for this to be static?